### PR TITLE
Actionview 5.1.4

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

File: packages/tfm-ror51-rubygem-concurrent-ruby/tfm-ror51-rubygem-concurrent-ruby.spec [hunk 1]
```diff
@@ -1,30 +1,31 @@
-# Generated from mini_portile2-2.3.0.gem by gem2rpm -*- rpm-spec -*-
```
Comment (ekohl) [inline packages/tfm-ror51-rubygem-concurrent-ruby/tfm-ror51-rubygem-concurrent-ruby.spec]: Was removing mini_portile2 intentional?

File: packages/tfm-ror51-rubygem-websocket-driver/tfm-ror51-rubygem-websocket-driver.spec [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+%global gem_name websocket-driver
+
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.6.5
+Release: 1%{?dist}
+Summary: WebSocket protocol handler with pluggable I/O
+Group:   Development/Languages
+License: MIT
+URL:     http://github.com/faye/websocket-driver-ruby
```
Comment (ekohl) [inline packages/tfm-ror51-rubygem-websocket-driver/tfm-ror51-rubygem-websocket-driver.spec]: https

File: packages/tfm-ror51-rubygem-websocket-extensions/tfm-ror51-rubygem-websocket-extensions.spec [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+%global gem_name websocket-extensions
+
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.2
+Release: 1%{?dist}
+Summary: Generic extension manager for WebSocket connections
+Group:   Development/Languages
+License: MIT
+URL:     http://github.com/faye/websocket-extensions-ruby
```
Comment (ekohl) [inline packages/tfm-ror51-rubygem-websocket-extensions/tfm-ror51-rubygem-websocket-extensions.spec]: https

File: packages/tfm-ror51-rubygem-concurrent-ruby/tfm-ror51-rubygem-concurrent-ruby.spec (hunk 1 above)
Comment (ehelms) [reply packages/tfm-ror51-rubygem-concurrent-ruby/tfm-ror51-rubygem-concurrent-ruby.spec]: Yes, it is not needed in production.

**Comment by ehelms** [comment]
The nice part of the commit currently is, its being removed in the same commit as the package that would normally require it but does not need it to actually build -- nokogiri.

**Comment by ekohl** [comment]
That's fair. 👍

**Pull request merged**